pkg/queue/manager.go

```diff
@@ -425,23 +441,27 @@ func (m *Manager) QueueInadmissibleWorkloads(ctx context.Context, cqNames sets.S
 	}
 }
 
-// queueAllInadmissibleWorkloadsInCohort moves all workloads in the same
+// queueAllInadmissibleWorkloadsInCohortFromCq moves all workloads in the same
 // cohort with this ClusterQueue from inadmissibleWorkloads to heap. If the
 // cohort of this ClusterQueue is empty, it just moves all workloads in this
 // ClusterQueue. If at least one workload is moved, returns true, otherwise
 // returns false.
 // The events listed below could make workloads in the same cohort admissible.
-// Then queueAllInadmissibleWorkloadsInCohort need to be invoked.
+// Then queueAllInadmissibleWorkloadsInCohortFromCq need to be invoked.
 // 1. delete events for any admitted workload in the cohort.
 // 2. add events of any cluster queue in the cohort.
 // 3. update events of any cluster queue in the cohort.
-func (m *Manager) queueAllInadmissibleWorkloadsInCohort(ctx context.Context, cq *ClusterQueue) bool {
-	if !cq.HasParent() {
-		return cq.QueueInadmissibleWorkloads(ctx, m.client)
+// 4. update of cohort.
+func (m *Manager) queueAllInadmissibleWorkloadsInCohortFromCq(ctx context.Context, cq *ClusterQueue) bool {
```

Comment:
This method's name is so long. Is there a possibility to make it shorter?

Reply:
Done.